nautobot_design_builder/models.py

```diff
@@ -368,11 +373,56 @@ def log(self, model_instance):
             entry.changes.update(model_instance.metadata.changes)
             entry.save()
         except ChangeRecord.DoesNotExist:
+            # Default full_control for created objects
```

Comment:
Please add a detailed code comment here explaining this change. It seems complex and I'm not sure what the intent is.